client/src/containers/home/index.js

```diff
@@ -2,15 +2,14 @@ import React from 'react'
 import { push } from 'react-router-redux'
 import { bindActionCreators } from 'redux'
 import { connect } from 'react-redux'
+import Map from '../map'
 
 const Home = props => {
   const { changePage, message } = props
```

Comment:
Comment out until we need it to prevent linting errors

Reply:
Are you asking to comment out the `import Map` line? This doesn't create any errors for me.